testing/smoke-ide-test/src/smokeIdeTest/groovy/org/gradle/ide/sync/IsolatedProjectsGradleceptionSyncTest.groovy

```diff
@@ -37,10 +30,16 @@ class IsolatedProjectsGradleceptionSyncTest extends AbstractIdeSyncTest {
         ideXmxMb = 4096
 
         when:
-        ideaSync(IDEA_COMMUNITY_VERSION, gradleCheckout)
+        ideaSync(IDEA_COMMUNITY_VERSION)
 
         then:
-        fixture.assertHtmlReportHasNoProblems()
+        report.assertHtmlReportHasProblems {
+            totalProblemsCount = 12
+            withLocatedProblem("Gradle runtime", "Project ':declarative-dsl-core' cannot dynamically look up a property in the parent project ':'")
```

Comment:
`Gradle runtime` is not ideal here, it happens because of https://github.com/gradle/gradle/blob/0348e5c0a15230eba9ea2845c2e87f4b7ad8da8b/testing/internal-integ-testing/src/main/groovy/org/gradle/integtests/fixtures/configurationcache/ConfigurationCacheProblemsFixture.groovy#L426 doesn't support new types of traces, like `PropertyTrace`